### DISCUSSION TITLE
Command to reset someones /back location

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#5363] [Enhancement]
Can we please get a command to reset someones /back so they cant use their last back location. 
This is for people that abuse it or keep a back location somewhere to exploit later.

This then allows us to use the trigger events in CMI (Like logging out) to reset someones back location. 

So it would work like `/resetback <username> <reason>` Then that player is sent the reason once it happens, then they can be told when/why its happened.

**Comment by khaiethan** [comment]
I second this, it would be a very useful feature to have for many servers.

**Comment by CreatingWithKass** [comment]
This would be extremely helpful!!

**Comment by johenkel** [comment]
That would be an amazing tool for server moderators !

**Comment by Zrips** [comment]
Next update
